src/chocolatey.resources/helpers/functions/Install-ChocolateyPath.ps1

```diff
@@ -69,42 +72,38 @@ Get-ToolsLocation
 param(
   [parameter(Mandatory=$true, Position=0)][string] $pathToInstall,
   [parameter(Mandatory=$false, Position=1)][System.EnvironmentVariableTarget] $pathType = [System.EnvironmentVariableTarget]::User,
+  [parameter(Mandatory=$false, Position=2)][switch] $Force = $false,
   [parameter(ValueFromRemainingArguments = $true)][Object[]] $ignoredArguments
 )
-  Write-Debug "Running 'Install-ChocolateyPath' with pathToInstall:`'$pathToInstall`'";
-  $originalPathToInstall = $pathToInstall
+  Set-StrictMode -Version 2

```

Comment:
@jberezanski remove the PowerShell v2 engine feature and `#Requires -Version 2` fails. I've seen it before. Would love for the minimum to be true, but have not seen it implemented in that way.